src/Date/Local.fs

```diff
@@ -82,12 +84,14 @@ module Local =
                   Thursday = "Thu"
                   Friday = "Fri"
                   Saturday = "Sat"
-                  Sunday = "Sun" } }
+                  Sunday = "Sun" }
+              DefaultFormat = "d/M/yyyy"
+              FirstDayOfTheWeek = 1 }
           Time =
             { AM = "AM"
               PM = "PM" } }
 
-    let french =
+    let getFrench() =
```

Comment:
Should be keep as variable

Reply:
Same for all the translations

Reply:
So skip Alfonzo's comment about functions for now?
> If we're making breaking changes, maybe we could turn the language options (french, russian...) into functions instead of values.  ...

Reply:
Ah I forgot is comment...
Hum, I don't know but the API using functions is not user friendly here that's why I don't like it...

Reply:
In F# there's a trick to compile something as a function but make it look as value: adding a generic parameter, so you can use `let french<'t> =` instead.

[REPL](http://fable.io/repl/#?code=let%20foo%3C'T%3E%20%3D%205%0D%0Alet%20bar%20%3D%205%0D%0A%0D%0Aprintfn%20%22%25i%22%20foo%0D%0Aprintfn%20%22%25i%22%20bar%0D%0A&html=%3Chtml%3E%3Cbody%3E%3C%2Fbody%3E%3C%2Fhtml%3E9)

Reply:
Ah yes that's true we used that at the time in Fulma.

Then, I am much in favor of this implementation :)

Reply:
Done, check it please.

Totally unrelated, but changed the test command to run fable on free port `dotnet fable webpack --port free -- --config tests/webpack.config.js`. Test just hang there because another fable daemon was running, happened today like three times...